update/application.go

```diff
@@ -336,6 +348,56 @@ func deleteWorkerJob(name string, cfg *apps.Config) {
 	cfg.WorkerJobs = newJobs
 }
 
+func (job scheduledJob) applyUpdates(cfg *apps.Config) {
+	if job.Name == nil {
+		return
+	}
+
+	if job.Command != nil || job.Size != nil {
```

Comment:
not sure this still needs to happen. We check this again further down in the loop.